Updated prerequisites + connecting to GDN (#345)
Macrometacorp/docs: docs/collections/graphs/quickstart.md

- INBOUND: `_from` ← `_to`
- ANY: `_from` ↔ `_to`.

## Prerequisites

- Access to a [Macrometa account](https://auth.paas.macrometa.io/) with sufficient permissions to create streams.

> - Have your API Key ready. For more information, refer to [Create API Keys](../account-management/api-keys/create-api-keys.md).

Reviewer: ```suggestion
- An API key. For more information, refer to [Create API Keys](../account-management/api-keys/create-api-keys.md).
```

Minor change to make list grammatically consistent

> - Install the appropriate SDK. For more information, refer to [Install SDKs](../sdks/install-sdks.md).

Reviewer: ```suggestion
- Appropriate SDK installed. For more information, refer to [Install SDKs](../sdks/install-sdks.md).
```

Minor change to make list grammatically consistent

> ### Step 1. Connect to GDN

Reviewer: I would either remove "Step 1" or add "Step 2," "Step 3," etc. to the rest of the headers.

Reviewer 2: Check the other quickstarts and getting started topics. I think this is consistent with them.

Also, we need to rewrite the whole thing anyway.

Reviewer 2: Dallas, don't worry about the step numbers for now. James and I will deal with it when we test and rewrite this quickstart.

Reviewer: Yeah, after posting my last comment I saw we have other Quickstarts similarly formatted and never edited my comment.  We can make it consistent when we rewrite.

> The first step in using the Macrometa Global Data Network (GDN) is to establish a connection to a local region. When this code executes, it initializes the server connection to the region URL you specified. For more information about connecting to the GDN, refer to [Authentication](../account-management/auth/index.md).

Reviewer: ```suggestion
To use graphs with Macrometa Global Data Network (GDN), you must first establish a connection to a local region. When this code runs, it initializes the server connection to the region URL you specified. For more information about connecting to GDN, refer to [Authentication](../account-management/auth/index.md).
```

@diana-macrometa Have we had a discussion about referring to GDN as "GDN" vs "the GDN"?

Reviewer 2: Not really. I would think it depends on context.
